src/SFA.DAS.Payments.RequiredPayments.RequiredPaymentsService.Interfaces/SFA.DAS.Payments.RequiredPayments.RequiredPaymentsService.Interfaces.csproj

```diff
@@ -10,13 +10,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ServiceFabric" Version="6.5.664" />
-    <PackageReference Include="Microsoft.ServiceFabric.Actors" Version="3.4.664" />
-    <PackageReference Include="Microsoft.ServiceFabric.Data" Version="3.4.664" />
-    <PackageReference Include="Microsoft.ServiceFabric.Diagnostics.Internal" Version="3.4.664" />
-    <PackageReference Include="Microsoft.ServiceFabric.Services" Version="3.4.664" />
-    <PackageReference Include="SFA.DAS.Payments.DataLocks.Messages" Version="1.0.0" />
-    <PackageReference Include="SFA.DAS.Payments.EarningEvents.Messages" Version="0.1.123" />
+	<PackageReference Include="Microsoft.ServiceFabric" Version="11.0.1000" />
```

Comment:
Do we need a DevOps ticket / sub-task to update the ServiceFabric infrastructure to a new version? Can this be done just for the RequiredPayments application or does it need to be done for all of them?